### PR TITLE
SLIM-675 Renaming SpecificConfigurationObject

### PR DESCRIPTION
**Pull request by hannek24**
Renaming SpecificConfigurationObject to SpecificConfigurationObject -SLIM-675

### CODE REVIEW

File: osgp-protocol-adapter-dlms/src/test/java/com/alliander/osgp/dto/valueobjects/smartmetering/ActionDtoBuilder.java [hunk 1]
```diff
@@ -112,8 +112,8 @@ public SynchronizeTimeRequestDto makeSynchronizeTimeRequestDataDto() {
         return new SynchronizeTimeRequestDto(120, true);
     }
 
-    public GetConfigurationRequestDto makeGetConfigurationRequestDataDto() {
-        return new GetConfigurationRequestDto();
+    public GetAttributeValuesRequestDto makeGetConfigurationRequestDataDto() {
```
Comment (robinbakkeruscgi) [inline osgp-protocol-adapter-dlms/src/test/java/com/alliander/osgp/dto/valueobjects/smartmetering/ActionDtoBuilder.java]: makeGetConfigurationRequestDataDto() should be renamed

and on line 78 there is still a :
public SetConfigurationObjectRequestDataDto makeSetConfigurationObjectRequestDataDto() ...

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/infra/messaging/processors/GetSpecificConfigurationObjectRequestMessageProcessor.java [hunk 2]
```diff
@@ -18,7 +18,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
```
Comment (robinbakkeruscgi) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/infra/messaging/processors/GetSpecificConfigurationObjectRequestMessageProcessor.java]: This class should be renamed as well

File: osgp-protocol-adapter-dlms/src/test/java/com/alliander/osgp/dto/valueobjects/smartmetering/ActionDtoBuilder.java (hunk 1 above)
Comment (hannek24) [inline osgp-protocol-adapter-dlms/src/test/java/com/alliander/osgp/dto/valueobjects/smartmetering/ActionDtoBuilder.java]: Those two are part of other functionality. They're meant for setting the configuration object (ObisCode: 0-1:94.31.3.255)

File: osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/infra/messaging/processors/GetSpecificConfigurationObjectRequestMessageProcessor.java (hunk 2 above)
Comment (hannek24) [inline osgp-protocol-adapter-dlms/src/main/java/org/osgp/adapter/protocol/dlms/infra/messaging/processors/GetSpecificConfigurationObjectRequestMessageProcessor.java]: fixed

**Pull request merged**